framework/platform/bpl/uci/cfg/bpl_cfg.cpp

```diff
@@ -415,5 +415,21 @@ bool cfg_get_persistent_db_enable(bool &enable)
     return true;
 }
 
+bool cfg_get_clients_persistent_db_max_size(int &max_size)
+{
+    int retVal = -1;
+    if (cfg_get_prplmesh_param_int_default("clients_persistent_db_max_size", &retVal,
+                                           DEFAULT_CLIENTS_PERSISTENT_DB_MAX_SIZE) == RETURN_ERR) {
+        MAPF_ERR(
+            "cfg_get_clients_persistent_db_max_size: Failed to read clients-persistent-db-max-size "
+            "parameter\n");
```

Comment:
I don't think you really need the function name in the LOG message...
```suggestion
        MAPF_ERR("Failed reading clients-persistent-db-max-size parameter");
```

Reply:
accepted your suggestion - thanks